tests/nodes_test.h

```diff
@@ -0,0 +1,230 @@
+#ifndef LIBUAST_NODES_TEST_H_
+#define LIBUAST_NODES_TEST_H_
+
+
+#include <cstdbool>
+#include <cstdio>
+#include <cstring>
+
+#include <iostream>
+#include <string>
+#include <vector>
+
+extern "C" {
+
+#include <CUnit/Basic.h>
+
+#include "mock_node.h"
+#include "roles.h"
+#include "testing_tools.h"
+#include "uast.h"
+#include "uast_private.h"
+
+bool EqualNodes(const Nodes *n1, const Nodes *n2) {
+  if (NodesSize(n1) != NodesSize(n2)) {
+    return false;
+  }
+  for (int i = 0; i < NodesSize(n1); i++) {
+    if (NodeAt(n1, i) != NodeAt(n1, i)) {
+      return false;
+    }
+  }
+  return true;
+}
+
+void testUastNew() {
```

Comment:
Google conventions require functions to start with uppercase, but I don't know if here the testing framework defines the naming.

Reply:
I thought so, but apparently not.